articles/etherchannel-technology/index.md

```diff
@@ -0,0 +1,380 @@
+- On- interfaces in this mode do not exchange PAgp packets.
+- Desirable-interfaces in this mode start negotiation with other interfaces by sending PAgp packets.
+- Auto- here interfaces respond to PAgp packets but cannot start a negotiation with other interfaces.
+
+**NOTE:** For an ether channel to form the modes must be compatible on either side of the links. The table below shows mode combination that can form ether channel links.
+| S1 mode   |    S2 mode     | Etherchannel Formation |
+| :-------- | :------------: | ---------------------: |
+| On        |       On       |                    YES |
+| On        | Desirable/Auto |                     NO |
+| Desirable | Desirable/Auto |                    YES |
+| Auto      |   Desirable    |                    YES |
+| Auto      |      Auto      |                     NO |
+
+
+#### Link Aggregation Control Protocol (LACP)
+This protocol works as the PAgp protocol, but its open standard and facilitates configuration of ether channel in the multi-vendor environments. The protocol is not limited to cisco switches only; it allows for both active links and standby links. It works in three modes:
+- On- interfaces in this mode do not exchange LACP packets.
+- Active-interfaces in this mode initiate negotiation with other interfaces by sending LACP packets.
+- Passive- here interfaces respond to LACP packets received but cannot start a negotiation with other interfaces.
+
+The table below shows mode combination that can form ether channel links with LACP.
+
+| S1 mode |    S2 mode     | Etherchannel Formation |
+| :------ | :------------: | ---------------------: |
+| On      |       On       |                    YES |
+| On      | Active/Passive |                     NO |
+| Active  | Active/Passive |                    YES |
+| Passive |     Active     |                    YES |
+| Passive |    Passive     |                     NO |
+
+**NOTE:** `Active links` are the ether channel links currently transmitting the traffic. When one of the current active link goes down  the `Standby links` become active and takes over.
+
+### EtherChannel network configurations
+In this article, we will learn how to configure ether channel using the two protocols discussed above.
+
+Consider a network with 3 switches connected to each other and 3 PCs each  connected to a switch as shown:
+
+![EtherChannel Network](/engineering-education/etherchannel-technology/network-setup.jpg)
+
+We have to configure the network to use one logical link instead of the two fastEthernet(Fa) links connecting the switches.
+We shall use 3 port-channels:
+- Port-channel 1-link connecting S1 and S2 (Fa0/1, Fa0/2 ).
+- Port-channel 2-link connecting S2 and S3 (Fa0/3, Fa0/4 )
+- Port-channel 3-link connecting S1 and S3 (Fa0/5, Fa0/6 )
+
+We shall use LACP on port-channels 1 and 2 and PAgp on Channel 3 through the following steps;
+
+#### Basic Switch Configurations
```

Comment:
Show your steps i.e 
Step One - Basic Switch Configurations.
Step Two - Port-channel 2 configurations using LACP mode passive and active.

Reply:
hello @BrianaNzivu done with the changes